src/main/java/core/basesyntax/strategy/OperationStrategy.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.model.Operation;
+import core.basesyntax.service.amount.OperationHandler;
+
+public interface OperationStrategy {
+    OperationHandler getOperationImpl(Operation operation);
```

Comment:
```suggestion
    OperationHandler getOperationHandler(Operation operation);
```
Implementation of the operation?) -_-